index.html

```diff
@@ -24,13 +25,153 @@
+                <li><a href=""><button id="skill">css</button></a></li>
+                <li><a href=""><button id="skill">javaScript</button></a></li>
+            </ul>
+            <button class="normal-button">See Project</button>
+        </div>
+
+
+    </div>
+    <div class="card-works">
+        <img class="porfolio-card1" src="./images/Snapshoot Portfolio 3.png" alt="">
+    <div class="project-desc">
+        <p class="project-title">Tonic</p>
+        <div class="frame2">
+            <h3>CANOPY</h3>
+            <img src="./images/Counter.png" alt="">
+            <h4>Back End Dev</h4>
+            <img src="./images/Counter.png" alt="">
+            <h4>2015</h4> 
+        </div>
+         
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag">
+            <li><a href=""><button id="skill">html</button></a></li>
+            <li><a href=""><button id="skill">css</button></a></li>
+            <li><a href=""><button id="skill">javaScript</button></a></li>
+        </ul>
+        <button class="normal-button">See Project</button>
+    </div>
+
+
+</div>
+<div class="card-works">
+    <img class="porfolio-card1" src="./images/Snapshoot Portfolio 4.png" alt="">
+<div class="project-desc">
+    <p class="project-title">Multi-Post Stories</p>
+    <div class="frame2">
+        <h3>CANOPY</h3>
+        <img src="./images/Counter.png" alt="">
+        <h4>Back End Dev</h4>
+        <img src="./images/Counter.png" alt="">
+        <h4>2015</h4> 
+    </div>
+     
+    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tag">
+        <li><button id="skill">html</button></li>
+        <li><button id="skill">css</button></li>
+        <li><button id="skill">javaScript</button></li>
+    </ul>
+    <button class="normal-button">See Project</button>
+</div>
+
+
+</div>
+    </section>
+    <section class="about-intro">
+        <p class="first-intro">About <br>Myself</p>
+        <p class="second-intro">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="social">LET'S CONNECT</p>
+
+        <div class="social-icons">
+            <ul class="social-icons">
+                <li><a href=""><img src="./images/twitter.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/github.png" alt=""></a></li>
+                <li><a href=""><img src="./images/Linkedin icon.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/mail.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/Vector.svg" alt=""></a></li>
+            </ul>
+        </div>
+        <button class="resumebtn">Get my resume </button>
```

Comment:
- [OPTIONAL] Kindly consider giving this button an exact width so that it remained at same size even when the container is getting bigger to make our design looks nicer.